MAESTRO/scATAC_H5Process.py

```diff
@@ -24,63 +24,63 @@ def mtxtoh5_parser(subparsers):
     Add main function mtx-to-h5 argument parsers.
     """
 
-    workflow = subparsers.add_parser("mtx-to-h5", 
+    workflow = subparsers.add_parser("mtx-to-h5",
         help = "Convert 10X mtx format matrix to HDF5 format.")
     group_input = workflow.add_argument_group("Input files arguments")
-    group_input.add_argument("--type", dest = "datatype", default = "Peak", 
-        choices = ["Peak", "Gene"], 
+    group_input.add_argument("--type", dest = "datatype", default = "Peak",
+        choices = ["Peak", "Gene"],
         help = "Type of the count matrix (Peak for scATAC-seq and Gene for scRNA-seq). DEFAULT: Peak.")
-    group_input.add_argument("--matrix", dest = "matrix", default = "matrix.mtx", 
+    group_input.add_argument("--matrix", dest = "matrix", default = "matrix.mtx",
         help = "Location of .mtx formatted matrix file. DEFAULT: matrix.mtx.")
-    group_input.add_argument("--feature", dest = "feature", default = "features.tsv", 
+    group_input.add_argument("--feature", dest = "feature", default = "features.tsv",
         help = "Location of feature file (required for the format of 'mtx'). Features correspond to row indices of count matrix. "
         "If the type is Peak, please provide the peak bed file with 3 columns. "
         "If the type is Gene, each row should include gene name. DEFAULT: features.tsv.")
     group_input.add_argument("--gene-column", dest = "gene_column", default = 2, type = int,
         help = "If the type is 'Peak', please specify which column of the feature file to use for gene names. DEFAULT: 2.")
-    group_input.add_argument("--barcode", dest = "barcode", default = "barcodes.tsv", 
+    group_input.add_argument("--barcode", dest = "barcode", default = "barcodes.tsv",
         help = "Location of barcode file. Cell barcodes correspond to column indices of count matrix. DEFAULT: barcodes.tsv. ")
-    group_input.add_argument("--species", dest = "species", default = "GRCh38", 
-        choices = ["GRCh38", "GRCm38"], type = str, 
+    group_input.add_argument("--species", dest = "species", default = "GRCh38",
```

Comment:
I feel strange to see this argument as 'species' but actually referring to 'genome assembly versions'.